fxmanifest.lua

```diff
@@ -3,9 +3,14 @@ lua54 'on'
 game 'gta5'
 ui_page 'html/index.html'
 
-shared_script '@renzu_shield/init.lua'
+shared_scripts {
+	--'@renzu_shield/init.lua',
+    '@ox_lib/init.lua',
+}
+
 server_scripts {
-	'@mysql-async/lib/MySQL.lua', -- uncomment if ghmatti and oxmysql
+	--'@mysql-async/lib/MySQL.lua', -- uncomment if ghmatti and oxmysql
+	'@oxmysql/lib/MySQL.lua'
```

Comment:
Without a comma at the end of this line, this fxmanifest file is broken and the resource simply won't load.

Reply:
Read it correctly. have comma before the comment

Also tell you that actually oxmysql has support for mysql-assync (This is achieved with the "provider").
Let's say it's like an alias

Reply:
This is in the middle of a list. 
`'config.lua',` comes immediately after this line, and without a comma it will be ignored by lua.
The syntax is simply invalid.

I don't know why you think the line isn't in the file, click "Files Changed" at the top of the PR and scroll down to this comment. It's there. You added it. This breaks the PR.

Reply:
Okay, I'm dumb because I was looking at renzuzu:main's code.
Change and thx